PerformanceTests/MallocBench/MallocBench/stress_aligned.cpp

```diff
@@ -148,7 +148,7 @@ void benchmark_stress_aligned(CommandLine&)
     
     srandom(1); // For consistency between runs.
 
-    size_t limit = 0x00001ffffffffffful;
+    unsigned long long limit = 0x00001ffffffffffful;
```

Comment:
I see that size_t was incorrect here, but how about:

```suggestion
    uint64_t limit = 0x00001ffffffffffful;
```

Reply:
Look nice, I will change it